backend/src/routes/doctorOrders.js

```diff
@@ -177,6 +177,14 @@ router.patch('/api/updateRx/:id/pickedUp', async (req, res) => {
       }
     });
     console.log('Sent RxFill to EHR and received status from EHR', status.data);
+
+    const remsAdminStatus = await axios.post('http://rems-administrator:8090/ncpdp', rxFill, {
```

Comment:
make sure to update this endpoint to match the ncpcp/script end point in rems admin.